src/shared/format-table.c

```diff
@@ -94,7 +96,7 @@ typedef struct TableData {
                 int percent;        /* we use 'int' as datatype for percent values in order to match the result of parse_percent() */
                 int ifindex;
                 union in_addr_union address;
-                /* … add more here as we start supporting more cell data types … */
+                sd_id128_t uuid;
```

Comment:
hmm, the very same change is in the "repart" PR in #14368. Any chance we can get that merged first?

Reply:
Obviously. I'll rebase this after your PR is merged.